src/main/launcher/main.c

```diff
@@ -304,6 +321,9 @@ int main(int argc, const char **argv)
     }
 
     boot_property_free(app_config);
+    if (bootstrap_config) {
+        boot_property_free(bootstrap_config);
+    }
```

Comment:
Why is this free'd so late?

Reply:
Because `app_config` can come from the bootstrap config (`component/param` node).